### PR TITLE
Fix CreateAedtApplication argtypes for 24.1

### PR DESCRIPTION
**Pull request by myoung301**
A change was made to the arguments to CreateAedtApplication for 24.1 so that the port can be a port (int) or object.

### CODE REVIEW

File: pyaedt/generic/grpc_plugin_dll.py
Comment (myoung301) [inline pyaedt/generic/grpc_plugin_dll.py:62]: @maxcapodi78 I'm not sure if there is a better way to get the version information here in the plugin initialization, but we need some way to handle the difference in argument types between 23R2 and 24R1.

Comment (myoung301) [reply pyaedt/generic/grpc_plugin_dll.py:62]: @Samuelopez-ansys - can you review and merge this PR if it looks good to you? It's holding up some related development.

Comment (myoung301) [reply pyaedt/generic/grpc_plugin_dll.py:62]: @jsalant22 Maybe you can take a look at this PR? Thanks!

**Review by jsalant22 — APPROVED** [review]

**Pull request merged**
